app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
-  # before_action :current_user
```

Comment:
I wondered if this was necessary(turns out it was not!). Thanks for taking a closer look at it and pulling out.